spec/models/video_spec.rb

```diff
@@ -8,33 +8,27 @@
 end
 
 describe "search by title" do
+  let(:futurama) { Fabricate(:video, title:"Futurama") }
+  let(:back_to_future) { Fabricate(:video, title: "Back to the Future") }
+  
   it "returns an empty array if there is no match" do
-    futurama = Video.create(title:"Futurama", description: "Space travel")
-    back_to_future = Video.create(title: "Back to the Future", description: "Time travel")
     expect(Video.search_by_title("Hello")).to eq([])
   end
 
   it "returns an array of one video for an exact match" do
-    futurama = Video.create(title:"Futurama", description: "Space travel")
-    back_to_future = Video.create(title: "Back to the Future", description: "Time travel")
     expect(Video.search_by_title("Futurama")).to eq([futurama])
   end
 
   it "returns an array of one video for a partial match" do
-    futurama = Video.create(title:"Futurama", description: "Space travel")
-    back_to_future = Video.create(title: "Back to the Future", description: "Time travel")
     expect(Video.search_by_title("rama")).to eq([futurama])
   end
 
   it "returns an array of all matches ordered by created_at" do
```

Comment:
The problem with his is, I can not see the created_at of the other item. You only set 1 - you should set the other to make it very clear if you want to test on that.